README.md

```diff
@@ -137,6 +137,37 @@ One can run MeTTa script from command line:
 metta ./tests/scripts/<name>.metta
 ```
 
+### Logger
+
+You can enable logging by prefixing the `metta` command line by
+
+```
+RUST_LOG=hyperon[::COMPONENT]*=LEVEL
+```
+
+where
+- `[::COMPONENT]*` is a, possibly empty, sequence of modules and
+  submodules of hyperon, such as `::metta`, `::metta::runner`,
+  `::common`, `::common::multitrie`, etc.
+- `LEVEL` is the log level.  Possible log levels are: `error`, `warn`,
+  `info`, `debug` and `trace`.
+
+For example, to log all hyperon messages at the `debug` level and
+below, while running `script.metta`, you may type:
+
+```
+RUST_LOG=hyperon=debug metta script.metta
+```
```

Comment:
Wait, did that trigger a test?

Reply:
No Nil, the reason is a non-compatible change in Rust toolchain. @luketpeterson already fixed it in #362

Reply:
All PRs trigger the test suite.  The issue was that the upgrade to Rust toolchain 1.71 broke the existing code.  But it's fixed in main now.

I merged main into your branch, and I expect the tests will pass now.